syfertext/string_store.py

```diff
@@ -1,3 +1,4 @@
+#from .typecheck.typecheck import type_hints
```

Comment:
import commented!

Reply:
Yeah there was an issue with stringstore.py, so I have removed the import for now